src/keboola/component/interface.py

```diff
@@ -952,6 +952,9 @@ def write_manifest(self, io_definition: Union[dao.FileDefinition, dao.TableDefin
         if not legacy_manifest:
```

Comment:
Tohle cely bych dal do separe funkce pro detekci manifest typu kdyz uz je tam tolik podminek

Reply:
a dalsi vec Storage uz zpracovavame je to `TableOutputMapping` tam by se mel pridat ten `data_type_support`
a pak by se to dalo napsat hezceji: 
```python
legacy_manifest = self.configuration.tables_output_mapping.data_type_support or self.environment_variables.data_type_support
```

Reply:
Jeste premyslim jak je tohle teda vymysleny? Co bude v `tables_output_mapping.data_type_support` pro existujici konfigurace. Tam prece nic neulozime pokud to user neulozi. Takze tam bude `null` ne? A potom `tables_output_mapping.data_type_support` bude vlastne vyplneny pokazdy. Ale jenom pro tu prvni vstupni komponentu, takze jak maji poznat processory co jsou dal, ze se celej ten chain ma chovat jako `none` ??